projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -1066,46 +1166,79 @@ function register_settings() {
 		add_settings_field( 'carousel_display_comments', __( 'Comments', 'jetpack' ), array( $this, 'carousel_display_comments_callback' ), 'media', 'carousel_section' );
 		register_setting( 'media', 'carousel_display_comments', array( $this, 'carousel_display_comments_sanitize' ) );
 
-		// No geo setting yet, need to "fuzzify" data first, for privacy
-		// add_settings_field('carousel_display_geo', __( 'Geolocation', 'jetpack' ), array( $this, 'carousel_display_geo_callback' ), 'media', 'carousel_section' );
-		// register_setting( 'media', 'carousel_display_geo', array( $this, 'carousel_display_geo_sanitize' ) );
 	}
 
-	// Fulfill the settings section callback requirement by returning nothing
-	function carousel_section_callback() {
-		return;
+	/**
+	 * Fulfill the settings section callback requirement by returning nothing.
+	 */
+	public function carousel_section_callback() {
 	}
 
-	function test_1or0_option( $value, $default_to_1 = true ) {
-		if ( true == $default_to_1 ) {
-			// Binary false (===) of $value means it has not yet been set, in which case we do want to default sites to 1
+	/**
+	 * Tests if a value is set
+	 *
+	 * @param mixed $value The value passed into this function with which to test.
+	 * @param bool  $default_to_1 Default is true.
+	 *
+	 * @return bool
+	 */
+	public function test_1or0_option( $value, $default_to_1 = true ) {
+		if ( true === $default_to_1 ) {
+			// Binary false (===) of $value means it has not yet been set, in which case we do want to default sites to 1.
```

Comment:
Since we're touching this line anyway, we should correct the incorrect terminology in the comment.
```suggestion
			// Boolean false (===) of $value means it has not yet been set, in which case we do want to default to 1.
```

Reply:
Got it, thanks!